app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -42,49 +63,51 @@ export default class MoreConfig extends Component {
         </small>
       );
     }
-    if (passedData.published) {
+    if (published) {
       publishedField = (
         <div>
           <h4>Danger Zone</h4>
-          <button onClick={onSaveDraft}>Unpublish Post</button>
+          <button type="button" onClick={onSaveDraft}>
+            Unpublish Post
+          </button>
         </div>
       );
     }
     return (
       <div className="articleform__overlay">
         <h3>Additional Config/Settings</h3>
         <button
+          type="button"
           className="articleform__exitbutton"
           data-content="exit"
           onClick={onExit}
         >
           ×
         </button>
-        <div>
-          <label>Canonical URL</label>
-          <input
-            type="text"
-            value={passedData.canonicalUrl}
-            name="canonicalUrl"
-            onKeyUp={this.props.onConfigChange}
-          />
-        </div>
+        {TextField({
+          label: 'Canonical URL',
+          id: 'canonicalUrl',
+          value: canonicalUrl,
+          onKeyUp: onConfigChange,
+        })}
         <small>
-          Change meta tag <code>canonical_url</code> if this post was first
-          published elsewhere (like your own blog)
+          Change meta tag
+          <code>canonical_url</code>
+          if this post was first published elsewhere (like your own blog)
```

Comment:
By changing this you lose the space between the word `tag` and the word `canonical_url`:

![Screenshot_2019-10-10 New Post - DEV(local) Community 👩‍💻👨‍💻(1)](https://user-images.githubusercontent.com/146201/66606198-e2e66700-ebb1-11e9-9a6b-832a4304e19c.png)

this is how it looked before:

![Screenshot_2019-10-10 New Post - DEV Community 👩‍💻👨‍💻](https://user-images.githubusercontent.com/146201/66606239-f265b000-ebb1-11e9-814a-104562e5cb8f.png)

Reply:
Oops, I didn't expect this behaviour. Changes Updated @rhymes